py/server/deephaven/parquet.py

```diff
@@ -248,6 +254,7 @@ def write(
         max_dictionary_keys (Optional[int]): the maximum dictionary keys allowed, if not specified, defaults to 2^20 (1,048,576)
         max_dictionary_size (Optional[int]): the maximum dictionary size (in bytes) allowed, defaults to 2^20 (1,048,576)
         target_page_size (Optional[int]): the target page size in bytes, if not specified, defaults to 2^20 bytes (1 MiB)
+        generate_metadata_files (Optional[bool]): whether to generate _metadata and _common_metadata files, default is False
```

Comment:
1. Default value.
2. This docstring is not useful to a user.  It needs to explain what these files are.